Serato-Now-Playing/serato_now_playing.py

```diff
@@ -605,7 +547,12 @@ def on_savebutton_clicked(self):
         self.upd_conf()
         self.close()
         self.errLabel.setText('')
-        TRAY.action_mixmode.setText('Active')
+        MIXMODE = 'active'
+        if CONFIG.local:
+            TRAY.action_mixmode.setText('Using: Oldest')
+        else:
+            MIXMODE = 'passive'
+            TRAY.action_mixmode.setText('Using: Newest')
```

Comment:
This whole interaction isn't working the way I want it to. When remote is active, it should not be possible to pick active mode.  It is _always_ passive.  So need to work that out.

Reply:
(it may be worthwhile to just commit as-is since this PR is already pretty big and has a LOT of really nice improvements and bug fixes in it already.)